recipes/faunus/conda_build_config.yaml

```diff
@@ -0,0 +1,18 @@
+python:
+    - 3.6
+    - 3.7 
+mpi:
+    - mpich
+    - nompi
+mpich: 3.2.*
+channel_sources:
+    - conda-forge/label/gcc7,defaults  # [linux]
+    - conda-forge,defaults             # [not linux]
+channel_targets:
+    - conda-forge gcc7                 # [linux]
+    - conda-forge main                 # [not linux]
+c_compiler:    # [linux]
+    - gcc      # [linux]
+cxx_compiler:  # [linux]
+    - gxx      # [linux]
+
```

Comment:
Someone else should comment on this. Maybe @minrk?

Reply:
This being the whole config. It looks OK to me, but IDK the mpi stuff very well.

Reply:
MPI handling now reflects the suggested changes.